### ISSUE TITLE
Screenreaders unable to use arrow navigation when focussing tabs

### ISSUE THREAD

**Issue by NMedlockGraniucs** [chakra-ui/chakra-ui#8809]
### Description

**Given** I am using a screen reader
**And** I have horizontal tabs setup
**When** I activate a tab
**And** use arrow down
**Then** I should be read the next piece of content

### Link to Reproduction

https://v2.chakra-ui.com/docs/components/tabs/usage

### Steps to reproduce

1. Activate NVDA screen reader
2. Click in the page above one of the example tab implementations
3. Tab to the First Tab
4. press the down arrow and up arrow
5. No change will be seen
6. Shift tab out of the element and use down arrow
7. See that you can down arrow past content if you don't focus the tabs

### Chakra UI Version

2.8.2

### Browser

Edge 128.0.2739.54

### Operating System

- [ ] macOS
- [X] Windows
- [ ] Linux

### Additional Information

Using NVDA screen reader 

https://github.com/chakra-ui/chakra-ui/blob/969fe57c97540cafa736ff64cfdff970c948c810/packages/components/src/tabs/use-tabs.ts#L240
Seems to be what is preventing these keys

**Comment by segunadebayo** [comment]
This issue has been automatically closed due to inactivity. If you believe this closure was in error, please reopen the issue or add a comment. Thanks for your understanding.

Note: This is an automated message.

**Comment by NMedlockGraniucs** [comment]
This still is occurring, should be a minor change to prevent default for the direction that is currently in use rather than always taking the up, down, left and right
